app/views/items/_item_grid.html.erb

```diff
@@ -0,0 +1,20 @@
+<div id="items">
+  <div class="container">
+      <div class="row ">
+        <% @items.each do |item| %>
+          <div class="col-sm-3 mb-4">
+          <div class="card">
+            <img class="card-img-top" src="https://via.placeholder.com/400x300" alt="Card image cap">
+            <div class="card-body text-center">
+              <h5 class="card-title text-truncate"><%= item.name %></h5>
+              <p class="card-text text-truncate">von: Alan Must</p>
+              <p class="card-text text-truncate text-success">verf&uuml;gbar</p>
```

Comment:
- incomplete: same goes here, but in this case we do have the data with the `#lendable?` operator; to fix it should be just a simple ternary like `<?= item.lendable? ? 'verfügbar' : 'verliehen' ?>` (might require consequential changes due to `#borrowed?` mentioned in the review comment above)